ember-app/app/models/new/board.js

```diff
@@ -92,6 +92,16 @@ var Board = Model.extend({
       return combined;
     }
   }),
+  issuesById: function(){
+    return _.groupBy(this.get('issues'), (issue)=> {
+      return issue.get('id');
+    });
+  }.property('issues.[]'),
```

Comment:
You bet, basically everytime you ask for the value of `issuesById` Ember will check the `issues.[]` array and ensure it has not changed otherwise it recomputes.